Source/src/modules/ModuleTexture.h

```diff
@@ -55,5 +69,7 @@ namespace Hachiko
         const short devil_version = IL_VERSION;
         static unsigned int LoadImg(const char* path, bool flip = true);
         static void DeleteImg(unsigned& img_id);
+
+        FT_Library freetype_lib;
```

Comment:
Should be a pointer?

Reply:
Nope, it seems to internally manage the instance